releasenotes/notes/ecs-resource-tags-replace-colon-28c50aa903be8b0e.yaml

```diff
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    The Agent can be configured to replace colon `:` characters in the ECS resource tag keys by underscores `_`.
+    This can be done by enabling `ecs_resource_tags_replace_colon: true` in the Agent config file
```

Comment:
formatting:

```suggestion
    The Agent can be configured to replace colon ``:`` characters in the ECS resource tag keys by underscores ``_``.
    This can be done by enabling ``ecs_resource_tags_replace_colon: true`` in the Agent config file
```